src/main/java/seedu/mark/ui/FolderStructureTreeView.java

```diff
@@ -42,9 +42,6 @@ public FolderStructureTreeView(FolderStructure folderStructure,
         populateTreeWithBookmarks();
```

Comment:
@openorclose Not sure whether the changes to this file is correct or not. With the original code, when I run `delete 1` followed by `undo`, the bookmark data is correct but the folder tree view is wrong. Not sure how the changes magically fix it :P

Reply:
Is it because when you clone the folder structure you clear everything first but the changes are not handled here?